cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -3610,23 +3610,44 @@ pMustCheckHash
             ]
       ]
 
-proposalHashCheckInfo :: HashCheckParamInfo ProposalUrl
-proposalHashCheckInfo =
-  HashCheckParamInfo
-    { flagSuffix = "anchor-data"
-    , dataName = "proposal"
-    , hashParamName = "--anchor-data-hash"
-    , urlParamName = "--anchor-url"
-    }
-
-constitutionHashCheckInfo :: HashCheckParamInfo ConstitutionUrl
-constitutionHashCheckInfo =
-  HashCheckParamInfo
-    { flagSuffix = "constitution-hash"
-    , dataName = "constitution"
-    , hashParamName = "--constitution-hash"
-    , urlParamName = "--constitution-url"
-    }
+pPotentiallyCheckedAnchorData
+  :: Parser (MustCheckHash anchorDataType)
+  -> Parser (L.Anchor L.StandardCrypto)
+  -> Parser (PotentiallyCheckedAnchor anchorDataType)
+pPotentiallyCheckedAnchorData mustCheckHash anchorData =
+  PotentiallyCheckedAnchor
+    <$> anchorData
+    <*> mustCheckHash
+
+pMustCheckProposalHash :: Parser (MustCheckHash ProposalUrl)
+pMustCheckProposalHash =
+  pMustCheckHash
+    HashCheckParamInfo
```

Comment:
This has to be fixed on `main` first.

Reply:
Won't merging this PR fix it in main?

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/921